sbatches/farmshare/singularity-jupyterlab.sbatch

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash  
+
+# Usage
+
+# 1. Default Jupyter notebook (with your scratch to work in)
+# $ bash start.sh singularity-jupyter
+
+# 2. Default Jupyter notebook with custom working directory
+# $ bash start.sh singularity-jupyter /scratch/users/<username>
+
+# 3. Select your own jupyter container on Sherlock!
+# $ bash start.sh singularity-jupyter /scratch/users/<username> /path/to/container
+
+# 4. Or any singularity container...
+# $ bash start.sh singularity /path/to/container <args>
+
+PORT=$1
+NOTEBOOK_DIR=/farmshare/scratch/users/$USER
+#CONTAINER=/farmshare/home/classes/bioe/301p/ce/ce
```

Comment:
I don't see the container defined here? This should be a variable and a default like the other file.

Reply:
Fixed on 173023f